### PR TITLE
Report SDK size impact on minimal app

### PR DESCRIPTION
**Pull request by lemnik**
## Goal
Report the size impact of the performance SDK on an app

## Testing
Danger should report the size of the SDK on this PR

### CODE REVIEW

File: features/fixtures/minimalapp/.gitignore [hunk 1]
```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
```
Comment (twometresteve) [inline features/fixtures/minimalapp/.gitignore:9]: Any reason not to ignore the whole `.idea` folder?

File: features/fixtures/minimalapp/Dangerfile [hunk 2]
```diff
@@ -0,0 +1,52 @@
+require 'fileutils'
+
+STANDALONE_HDPI_BASE = "standalone-hdpi.apk"
+STANDALONE_ARM64_V8A = "standalone-arm64_v8a_hdpi.apk"
+STANDALONE_ARMEABI_V7A = "standalone-armeabi_v7a_hdpi.apk"
+STANDALONE_X86_64 = "standalone-x86_64_hdpi.apk"
+STANDALONE_X86 = "standalone-x86_hdpi.apk"
```
Comment (twometresteve) [inline features/fixtures/minimalapp/Dangerfile:7]: Are these used?

**Review by twometresteve — APPROVED** [review]

File: .idea/compiler.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="11" />
+    <bytecodeTargetLevel target="17" />
```
[the remark this answers is not in the document]
Comment (lemnik) [reply .idea/compiler.xml]: It looks like that got in before our standard `.gitignore` was added. Will remove

File: features/fixtures/minimalapp/Dangerfile (hunk 2 above)
Comment (lemnik) [reply features/fixtures/minimalapp/Dangerfile]: Not anymore, thanks for the spot!

**Pull request merged**